lib/bulbs-elements/util/prep-ga-event.js

```diff
@@ -0,0 +1,18 @@
+export default function prepGaEvent (gaPrefix, gaId, dimensions = {}) {
```

Comment:
I feel like the name of this function and file is misleading because you're prepping a GA tracker, not an event.  The GA tracker can do more then just sending events.  Reading this makes me think this should be `prepGaTracker` and `prep-ga-tracker.js`